TestsCalculator/SettingsPageTests.cs

```diff
@@ -0,0 +1,199 @@
+        public void CheckCurrencyFormatOptions()
+        {
+            //Arrange
+            SettingsPage settingsPage = new SettingsPage(driver);
+            List<string> expectedCurrencyFormat = new List<string> { "$ - US dollar", "€ - euro", "£ - Great Britain Pound" };
+
+            //Act
+            SelectElement s = settingsPage.CurrencyFormatOptions;
+            IList<string> actualCurrencyFormat = new List<string>();
+            for (int j = 0; j < s.Options.Count; j++)
+            {
+                actualCurrencyFormat.Add(s.Options.ElementAt(j).Text);
+            }
+
+            //Assert
+            Assert.AreEqual(expectedCurrencyFormat, actualCurrencyFormat);
+        }
+
+        [Test]
+        public void CancelBtnOpensDepositPage()
+        {
+            //Arrange
+            SettingsPage settingsPage = new SettingsPage(driver);
+
+            //Act
+            settingsPage.CancelBtn.Click();
+
+            //Assert
+            string actualUrl = driver.Url;
+            string expectedUrl = "http://127.0.0.1:8080/Deposit";
+            Assert.AreEqual(expectedUrl, actualUrl);
+        }
+
+        [TestCase("MM dd yyyy", "M dd yyyy")]
+        [TestCase("MM/dd/yyyy", "M/dd/yyyy")]
+        [TestCase("dd-MM-yyyy", "dd-M-yyyy")]
+        [TestCase("dd/MM/yyyy", "dd/M/yyyy")]
+        public void SaveDateFormat(string dateFormat, string dateDisplay)
+        {
+            //Arrange
+            SettingsPage settingsPage = new SettingsPage(driver);
+            CalculatorPage calculatorPage = new CalculatorPage(driver);
+
+            //Act
+            settingsPage.DateFormatOptions.SelectByText(dateFormat);
+            settingsPage.SaveBtn.Click();
+            IAlert alert = driver.SwitchTo().Alert();
+            alert.Accept();
+
+            //Assert
+            Assert.AreEqual(DateTime.Today.ToString(dateDisplay), calculatorPage.EndDate);
+        }
+
+        [TestCase("123 456 789,00", "100 000,00")]
+        [TestCase("123 456 789.00", "100 000.00")]
+        [TestCase("123.456.789,00", "100.000,00")]
+        [TestCase("123,456,789.00", "100,000.00")]
+        public void SaveNumberFormat(string number, string income)
+        {
+            //Arrange
+            SettingsPage settingsPage = new SettingsPage(driver);
+            CalculatorPage calculatorPage = new CalculatorPage(driver);
+
+            //Act
+            settingsPage.NumberFormatOptions.SelectByText(number);
+            settingsPage.SaveBtn.Click();
+            IAlert alert = driver.SwitchTo().Alert();
+            alert.Accept();
+            //Fill data to verify
+            calculatorPage.AmountField.SendKeys("100000");
+
+            // Assert
+            Assert.AreEqual(income, calculatorPage.Income);
+        }
+
+        [TestCase("£ - Great Britain Pound", "£")]
+        [TestCase("€ - euro", "€")]
+        [TestCase("$ - US dollar", "$")]
+        public void SaveCurrencyFormat(string currency, string currencySymbol)
```

Comment:
It's a good rule to add 'Test' to the end of the name of each test.

```suggestion
        public void SaveCurrencyFormatTest(string currency, string currencySymbol)
```